mhs/outbound/outbound/request/synchronous/request_body_schema.py

```diff
@@ -0,0 +1,53 @@
+import dataclasses
+from typing import List
+
+import marshmallow.validate
+
+# These allowed content types for attachments are taken from the EIS part 2.5.4.2
+_ATTACHMENT_ALLOWED_CONTENT_TYPES = (
+    'text/plain', 'text/html', 'application/pdf', 'text/xml', 'application/xml', 'text/rtf', 'audio/basic',
+    'audio/mpeg', 'image/png', 'image/gif', 'image/jpeg', 'image/tiff', 'video/mpeg', 'application/msword'
+)
+
+
+@dataclasses.dataclass
+class Attachment:
+    is_base64: bool
+    content_type: str
+    payload: str
+
+
+class AttachmentSchema(marshmallow.Schema):
+    is_base64 = marshmallow.fields.Bool(required=True,
+                                        description='Whether the attachment payload is base64-encoded or not. This is '
+                                                    'only required for binary attachments eg images.',
+                                        truthy={True}, falsy={False})
+    content_type = marshmallow.fields.Str(required=True, description='Content type of the attachment',
+                                          validate=marshmallow.validate.OneOf(_ATTACHMENT_ALLOWED_CONTENT_TYPES))
+    payload = marshmallow.fields.Str(required=True,
+                                     description='The attachment, possibly base64-encoded as per is_base64.',
+                                     validate=marshmallow.validate.Length(min=1))
+    description = marshmallow.fields.Str(required=True, description='Description of the attachment',
+                                         validate=marshmallow.validate.Length(min=1))
```

Comment:
@colin-bateson Should there be a max length for description? I can't see anything written down in the  EIS appendix A.1.1.4 and A.1.1.5 .

Reply:
Can you add a comment to the ticket to say that we are not implementing a max length, as this is not defined in the EIS?

Reply:
I looked again at the EIS and found some docs around max length for attachments but not description or payload (the HL7 payload). So I've put a comment onto [the story](https://gpitbjss.atlassian.net/browse/RT-107).